packages/cli/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@lightdash/cli",
+  "version": "0.1.5",
+  "license": "MIT",
+  "bin": {
+    "lightdash": "dist/index.js"
+  },
+  "dependencies": {
+    "commander": "^9.2.0"
+  },
+  "description": "Lightdash CLI tool",
+  "devDependencies": {
+    "@types/node": "^14.14.35",
+    "@typescript-eslint/eslint-plugin": "^5.22.0",
+    "@typescript-eslint/parser": "^5.22.0",
+    "eslint": "^8.7.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^16.1.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-json": "^3.0.0",
+    "prettier": "2.3.2",
+    "prettier-plugin-organize-imports": "^2.3.4",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.2.3"
+  },
+  "scripts": {
+    "build": "tsc --build tsconfig.json",
+    "lint": "eslint ./src",
+    "format": "prettier --check --ignore-unknown ./src"
```

Comment:
We need to update the commands in the main package.json

Reply:
you mean adding `ci-format` or `ci-lint` do you actually use those ?

Reply:
In the main package.json we have the following comamnds

```
    "lint": "run-s common-lint backend-lint frontend-lint e2e-lint",
    "format": "run-s common-format backend-format frontend-format e2e-format",
```

These commands are used in the github action `.github/workflows/test-and-build.yml` that is used to check PR. 

```
    - name: Lint all packages
      run: yarn lint
    - name: Check formatting all packages
      run: yarn format

    - name: Build common
      run: yarn common-build

    - name: Test backend
      run: yarn workspace backend test
```